website/content/apprenticeships/8th-light.md

```diff
@@ -0,0 +1,11 @@
+---
+company: "8th Light"
+link: "https://8thlight.com/apprenticeship/"
+location: 
+  - "London, UK"
+description: "8th Light is a software consultancy that is passionate about crafting beautiful and durable software products."
+draft: false
+---
+able software products."
```

Comment:
Looks like you got some extra stuff outside the front matter that can be removed